CHANGES.rst

```diff
@@ -9,6 +9,7 @@ Unreleased
 * Fix bug with positive ``pause_after`` values in streams provided by
   :meth:`.stream_generator` where the wait time was not reset after a yielded
   ``None``.
+* Improve url parsing in creation of `.Submission` objects
```

Comment:
Can you be more specific that it permits parsing short urls with trailing slash? Also can you use the following:

    :class:`.Submission`?

Finally, can you end the sentence with a period?

Reply:
Fixed.